src/commands/modping.ts

```diff
@@ -0,0 +1,167 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { ModPingSettings } from '@/models/modPingSettings';
+import { getRoleFromSettings } from '@/util';
+import type { ChatInputCommandInteraction, GuildMember, Role } from 'discord.js';
+
+async function handleToggle(interaction: ChatInputCommandInteraction, role: Role): Promise<string> {
+	const member = interaction.member as GuildMember;
```

Comment:
Don't really want to cast this unless we can prove it will always be the right type.

Can do
```ts
const member = await interaction.guild!.members.fetch(interaction.user.id);
```
So that you're always fetching the member so it should always be safe